src/Sylius/Bundle/CoreBundle/Migrations/Version20230331091850.php

```diff
@@ -16,30 +16,38 @@
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\Migrations\AbstractMigration;
 
-final class Version20230329093932 extends AbstractMigration
+final class Version20230331091850 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return 'Add ChannelPriceHistoryConfig';
+        return 'Create ChannelPricingLogEntry, ChannelPriceHistoryConfig, table of excluded taxons from showing the lowest price and add channel_price_history_config_id column to sylius_channel table';
```

Comment:
IMO it should be enough:
```suggestion
        return 'Initialize Price History';
```